src/components/SideBar/Sidebar.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from 'react';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
+import { RoleContext } from 'contexts/roleContext';
+import classes from './SideBar.module.scss';
+
+const sideBarLinks = [
+  { name: 'Overvirew', link: '/dashboard/rentalsoverview' },
+  { name: 'Books', link: '/dashboard/booksoverview' },
+  { name: 'Users', link: '/dashboard/users' }
+];
```

Comment:
If you use `sideBarLinks` object in many components you should put that in some folder like utils or something else and then import it in many components.

If you use it only in Sidebar component, it is ok to stay in that component.